languages/elixir/exercises/concept/charlists/.docs/introduction.md

```diff
@@ -0,0 +1,48 @@
+Charlists are created using single quotes.
+
+```elixir
+'hello'
+```
+
+A charlist is a list of integers where all the integers are valid code points.
+
+```elixir
+[65, 66, 67]
+# => 'ABC'
+```
+
+You can prepend a character with `?` to get its code point.
+
+```elixir
+?A
+# => 65
+
```

Comment:
```suggestion

# any list of integers whose decimal values are comprised of printable codepoints between 1 and 255 may be displayed as a charlist, e.g.
```

Reply:
At some point I think it may be important for students to understand why their lists of integers may be displayed as a charlist, or vice versa where their valid charlist is printed as a list of integers.

Reply:
I avoided mentioning that in `introduction.md` because this knowledge is not necessary to solve the exercise. I went crazy with the topic in `after.md`.